### ISSUE TITLE
[SUGGESTION] | Reworked Holy Wars Eligibility+ Isolationist Provocation

### ISSUE THREAD

**Issue by Jablains** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth#467]
**DO NOT REMOVE PRE-EXISTING LINES**
----------------------------------------------------------------------------------------------------------
**Describe your suggestion in full detail below:**

Many leaders that are pacifists/isolationists lorewise go to war frequently due to CK2 AI. Currently, any leader with zealous will declare holy wars regardless of their other traits. 

I propose changes that will make leader behaviour slightly more lore friendly.

- Have isolationist factions like Night Elves have a hidden modifier that prevents them from going to any war unless war is declared on them first, which them removes it. Joining grand alliances like the Alliance or Horde should remove it as well.

- Make Holy War requirement more stringent, requiring a big philosophy change like joining the Scarlet Crusade, cruel/impaler/cannibal traits or being a secret cultist. 

- Have a special tributary CB that targets "mob" counties in a de jure Kingdom (like Gnolls and Murlocs), simulating containment of a population of tribals that are hard to root out. Full removal would only come from a leader that is willing to ravage the land and commit total genocide, requiring traits qualifiying for my proposed more stringent Holy War CB. Losing land to these populations would remove CB restrictions, just like isolationists. Maybe have special events that force you to cede land to tribals if you are in debt.

Perhaps this can feed into a global event where "World of Warcraft" begins, unrestricting Casus Belli to regular levels for everyone, allowing the Thrall's Horde and neutrals time to consolidate.

**Comment by zumbak04** [comment]
> * Make Holy War requirement more stringent, requiring a big philosophy change like joining the Scarlet Crusade, cruel/impaler/cannibal traits or being a secret cultist.

I'm afraid it will make Holy Wars almost impossible for AI while player will know what to do to get CB.

**Comment by Jablains** [comment]
@zumbak04  You are right about Holy Wars. My intention was to emulate the situation in WoW with Gnolls, Murlocs etc holding on as a problem too big to ignore but too small for wars. It doesn't quite capture the same feeling if they get wiped out really quickly in Holy Wars, which they do in all of my games. 

What about having that increased restriction on Holy Wars apply only to "Good" religions versus pagans? They would still have unrestricted Holy War CBs against the Cult of the Damned etc. My proposed tributary CB will then be the default solution to "Mob" counties if the leader isn't the type to take a drastic/cruel solution to the problem. Perhaps, they will also be a rare periodic event where non-player counties can be taken over by mobs to simulate the problem, possibly with an event spawned adventurer taking over the county being a possible solution.

Sorry in advance if all this is just too complicated to implement.

**Comment by zumbak04** [comment]
Just a few notes:

> * Have isolationist factions like Night Elves have a hidden modifier that prevents them from going to any war unless war is declared on them first, which them removes it. Joining grand alliances like the Alliance or Horde should remove it as well. Perhaps western protectorate mechanics can be borrowed to faciliate an isolatinist/expansionist behaviour change.

This is partially implemented, the Kaldorei rulers prefer not to go beyond Kaldorei borders.

> Perhaps, they will also be a rare periodic event where non-player counties can be taken over by mobs to simulate the problem, possibly with an event spawned adventurer taking over the county being a possible solution.

And I like this idea. It may be good.
